src/Fantomas.Core/CodePrinter.fs

```diff
@@ -2019,7 +2019,14 @@ and genExpr astContext synExpr ctx =
             +> !- "] <- "
             +> autoIndentAndNlnIfExpressionExceedsPageWidthUnlessStroustrup (genExpr astContext) valueExpr
         | NamedIndexedPropertySet (sli, e1, e2) ->
+            let needsSep =
```

Comment:
I would have typically done this in one step:
```fsharp
let sep =
    match e1 with
    | SynExpr.Const _
    | SynExpr.Ident _ -> sepSpace
    | _ -> sepNone
```

`sepNone` is an alias for `id`, which we use to avoid any confusion when `ident` is abbreviated.

Reply:
Oh yeah, that's way nicer. Will keep that in mind.